postgres-matrix.yml

```diff
@@ -1,5 +1,5 @@
 name: Postgres Version Matrix
-project_name: azure-gdpr
```

Comment:
Would this cause issues in the rpm package names?

We would not want to break upgrades due to name changes here.

Reply:
Can you test that upgrades are ok after the change here?

Reply:
No it is not related

Reply:
BTW, there is no need to test since debian packages are new

Reply:
I was talking about rpm package upgrades. I agree that debian has no prior packages to upgrade from.

Reply:
You can check it here
https://github.com/citusdata/packaging/runs/7318282826?check_suite_focus=true
Rpm packages was not published since package names are the same.